packages/opds-web-client/src/components/Root.tsx

```diff
@@ -113,6 +119,12 @@ export class Root extends React.Component<RootProps, RootState> {
     let showBreadcrumbs = this.props.collectionData && breadcrumbsLinks.length > 0;
     let showSearch = this.props.collectionData && this.props.collectionData.search;
     let showFooter = this.props.collectionData && Footer;
+    // The tabs should only display if the component is passed and if
+    // the catalog is being displayed and not a book.
+    let showCollectionHeader = !!CollectionHeader && !showBook;
+    let facetGroups = this.props.collectionData ?
+      this.props.collectionData.facetGroups : [];
+    let rootUrl = this.props.collectionData ? this.props.collectionData.url : "";
```

Comment:
Looks like you didn't end up needing this variable.

Reply:
Removed it.